File: projects/order-creation-queries/core/src/main/java/com/robinfood/core/entities/menu/MenuSuggestedPortionDataEntity.java
package com.robinfood.core.entities.menu;

import lombok.Data;

@Data
public class MenuSuggestedPortionDataEntity {

    private final Long dicUnitId;
    private final Long id;
    private final String image;
    private final String name;
    private final Long parentId;
    private final Double quantity;
    private final String sku;
}

File: projects/posv2/core/src/main/kotlin/com/robinfood/core/entities/menu/MenuSuggestedPortionDataEntity.kt
package com.robinfood.core.entities.menu

open class MenuSuggestedPortionDataEntity(
        dicUnitId: Long?,
        id: Long?,
        image: String?,
        name: String?,
        parentId: Long?,
        quantity: Double?,
        sku: String?
) : MenuPortionEntity(id, dicUnitId, name, image, parentId, quantity, sku)